views/templates/admin/addons-suggestion.tpl

```diff
@@ -28,22 +28,21 @@
   </div>
   <div class="suggestion-category-details">
     <div>
-      {l s='To go further:' d='Modules.Productcomments.Shop'}
+      {l s='To go further:' d='Admin.Modules.Feature'}
```

Comment:
What about _Want to go further?_ in _Admin.Modules.Feature_, I think it is more engaging?

Reply:
You're right it sounds better, I will change it

Reply:
Done